app/src/main/java/mozilla/org/webmaker/javascript/WebAppInterface.java

```diff
@@ -82,6 +84,11 @@ public void logText(String txt){
         Log.v("wm", txt);
     }
 
+    @JavascriptInterface
+    public void goBack() {
+        mActivity.finish();
```

Comment:
Unfortunately I can't do that, since `onBackPressed` is being overridden, right?